daemon/openrazer_daemon/hardware/keyboards.py

```diff
@@ -699,6 +699,27 @@ class RazerCynosaChroma(_RippleKeyboard):
     DEVICE_IMAGE = "https://assets.razerzone.com/eeimages/support/products/1256/1256_cynosa_chroma.png"
 
 
+class RazerCynosaChromaV2(_RippleKeyboard):
+    """
+    Class for the Razer Cynosa Chroma V2
```

Comment:
Based on z3ntu's [comment](https://github.com/openrazer/openrazer/pull/1257/files#r495458959) I would think it needs to be updated here.